config/routes.rb

```diff
@@ -6,6 +6,9 @@
 
   resources :users
 
+  root 'search#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.